modules/ROOT/examples/live-demos/local-upload/index.js

```diff
@@ -7,11 +7,11 @@ tinymce.init({
   images_upload_url: 'postAcceptor.php',
 
   /* we override default upload handler to simulate successful upload*/
-  images_upload_handler: (blobInfo, success, failure) => {
-    setTimeout(() => {
+  images_upload_handler: (blobInfo) => {
+    return new Promise((resolve) => {
       /* no matter what you upload, we will turn it into TinyMCE logo :)*/
-      success('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png');
-    }, 2000);
+      resolve('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png');
+    });
   },
```

Comment:
This conversion is not correct... you've dropped the `setTimeout`. Also the `new Promise` part should be collapsed so it's on the same line, e.g. `(blobInfo) => new Promise(resolve) => {`